models/README.md

```diff
@@ -0,0 +1,16 @@
+# Models
+
+Please refer [training]() section to understand how our best model is trained
+
+There are **three** main models in this repository
```

Comment:
This can be confusing.. because there are 4 models, including semantic. Can you add a semantic as well and say sth like that is stored in hugging face and link to that store etc.

Reply:
That way, explaining Hybrid Model will be even better (since you can now refer to semantic

Reply:
you can even add a semantic.txt file here with a note inside that says, the semantic is stored in the hugging face store and add the link